estate/models/estate_property.py

```diff
@@ -0,0 +1,31 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+
+    name = fields.Char(string='Property Name', required=True)
+    description = fields.Text(string='Property Description')
+    postcode = fields.Char(string='Property Postcode')
+    date_availability = fields.Date(string='Availability', copy=False, default=fields.Date.add(fields.Date.today(),
+                                                                                               months=3))
+    expected_price = fields.Float(string='Expected Price', required=True)
+    selling_price = fields.Float(string='Selling Price', readonly=True, copy=False)
+    bedrooms = fields.Integer(string='Bedrooms', default=2)
+    living_area = fields.Integer(string='Living Area')
+    facades = fields.Integer(string='Facades')
+    garage = fields.Boolean(string='Garage')
+    garden = fields.Boolean(string='Garden')
+    garden_area = fields.Integer(string='Garden Area')
+    garden_orientation = fields.Selection(string='Garden Orientation', selection=[('north', 'North'),
+                                                                                  ('south', 'South'),
+                                                                                  ('east', 'East'),
+                                                                                  ('west', 'West')])
```

Comment:
With selection fields, a more readable indentation is the following:
```suggestion
    garden_orientation = fields.Selection(
        string='Garden Orientation', 
        selection=[
            ('north', 'North'),
            ('south', 'South'),
            ('east', 'East'),
            ('west', 'West'),
        ]
    )
```
This comment also applies for the selection field below